lib/ecto_enum.ex

```diff
@@ -79,6 +102,10 @@ defmodule EctoEnum do
           {:ok, @enum_map[int]}
         end
 
+        def load(string) when is_binary(string) do
+          {:ok, String.to_atom(string)}
```

Comment:
The performance difference is negligible. The handling of the pathological condition (as you pointed out) is much better with the look up.

This brings up an interesting point: what should we do if the value loaded from the database is not recognized by our table? Seems best to fail with `:error`.